src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/DefaultMonitorService.java

```diff
@@ -70,37 +72,50 @@ public DefaultMonitorService(Log log) {
 
   @Override
   public MonitorConnectionContext startMonitoring(
-      String node,
+      Set<String> nodeKey,
```

Comment:
singular names are usually not great for variables with lists/sets.
nodeKeys, or nodes perhaps? it's important to know what's being passed there.